includes/ddb_cp.admin.inc

```diff
@@ -7,7 +7,9 @@
 /**
  * Returns the main menu for the DDB administrative pages.
  *
- * @return
+ * @todo: It would be better to return and render array and have the theme layer call render!
```

Comment:
I has been agreed with Gitte Barlach and Rolf Madsen that we won't do any theming in the control panel.